src/Form/FormLabel.js

```diff
@@ -8,18 +8,18 @@ import { createStyleSheet } from 'jss-theme-reactor';
 import withStyles from '../styles/withStyles';
 
 export const styleSheet = createStyleSheet('MuiFormLabel', theme => {
-  const focusColor = theme.palette.primary[500];
+  const focusColor = theme.palette.primary[theme.palette.type === 'light' ? 'A700' : 'A200'];
```

Comment:
`A` colors are supposed to used only with the `.accent` color. Use `700` and `200` instead with the primary color.